### DISCUSSION TITLE
@types/express

### DISCUSSION THREAD

**Discussion by stempelo** [DefinitelyTyped/DefinitelyTyped#53898] [Issues with a @types package]
Passing from @types/express@4.17.11 to @types/express@4.17.12
there is a problem with this simple code (example expressjs)
....
app.use(express.json())   <---------- PROBLEM HERE
app.use(cors());

....

No overload matches this call.
  The last overload gave the following error.
    Argument of type 'NextHandleFunction' is not assignable to parameter of type 'PathParams'.
      Type 'NextHandleFunction' is missing the following properties from type '(string | RegExp)[]': pop, push, concat, join, and 27 more.

If i use @types/express@4.17.11 i don't have any problem

**Comment by vrumger** [comment]
As mentioned in #53807, it's related to `@types/node` v15.
